### PR TITLE
Restore integration test in fungible application

### PR DESCRIPTION
**Pull request by jvff**
# Motivation

The only integration test example that was in the repository was accidentally removed during the restructuring of the examples (#695).

# Solution

Restore the test from before it was removed.

### CODE REVIEW

**Review by ma2bd — APPROVED** [review]
Oops, I missed that in my review

**Review by ma2bd — APPROVED** [review]

File: examples/fungible/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,10 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 thiserror = { workspace = true }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+linera-sdk = { workspace = true, features = ["test", "wasmer", "wasmtime"] }
```
Comment (ma2bd) [inline examples/fungible/Cargo.toml:19]: why both VMs?

File: linera-sdk/src/test/integration/chain.rs [hunk 2]
```diff
@@ -360,11 +361,14 @@ impl ActiveChain {
     pub async fn query<Output>(
         &self,
         application_id: ApplicationId,
-        query: impl ToBcsBytes,
+        query: impl AsRef<str>,
```
Comment (ma2bd) [inline linera-sdk/src/test/integration/chain.rs:364]: I feel that this should be a `serde_json::Value`.

File: examples/fungible/Cargo.toml (hunk 1 above)
Comment (jvff) [reply examples/fungible/Cargo.toml]: No reason I think. It should work with any one of them :+1:

File: linera-sdk/src/test/integration/chain.rs (hunk 2 above)
Comment (jvff) [reply linera-sdk/src/test/integration/chain.rs:364]: I'm not sure if GraphQL queries are JSON though. I think they are custom strings, but we can change it later if it's not :thinking:

Comment (ma2bd) [reply linera-sdk/src/test/integration/chain.rs:364]: They are

Comment (ma2bd) [reply linera-sdk/src/test/integration/chain.rs:364]: Right. It's only the internal values I guess.

**Pull request merged**